charts/prometheus/values.yaml

```diff
@@ -491,57 +490,51 @@ prometheus-rules:
             for: "0m"
             keep_firing_for: "15m"
             description: "There are {{ $value }} EVM Nodes with chainId {{ $labels.chainId }} is down"
-          # Pubsub
-          Pubsub_subscription_unacknowledged_messages:
-            enabled: false
-            for: "5m"
-            description: "There are {{ $value }} unacknowledged messages (a.k.a. backlog messages) in a subscription '{{ $labels.subscription_id }}' for > 5min"
-            expr: max by (subscription_id) (stackdriver_pubsub_subscription_pubsub_googleapis_com_subscription_num_undelivered_messages) > 100
       WarningMetric:
-        enabled: true
+        enabled: false
         interval: "31s"
         rules:
           # redis
           Redis_disconnected_slaves:
-            enabled: true
+            enabled: false
             for: "15m"
             description: "Redis not replicating for all slaves. Consider reviewing the redis replication status."
             expr: count by (job) (redis_instance_info{role="slave"}) < 2
           # postgres
           Postgres_logical_backup_error:
-            enabled: true
+            enabled: false
             for: "2h"
             description: "Cronjob {{ $labels.cronjob }} failed"
             expr: ((kube_cronjob_next_schedule_time - on(cronjob, exported_namespace) kube_cronjob_status_last_successful_time)/3600) > 25
           # Resources
           PVC_low_capacity:
-            enabled: true
+            enabled: false
             for: "1h"
             description: "Volume {{ $labels.persistentvolumeclaim }} located in namespace {{ $labels.namespace }} (on {{ $labels.node }}) has capacity < 10%"
             expr: (kubelet_volume_stats_available_bytes / on(instance,namespace,node,persistentvolumeclaim,service) kubelet_volume_stats_capacity_bytes*100) < 10
           High_postgres_container_cpu_throttling:
-            enabled: true
+            enabled: false
             for: "1h"
             description: "Container {{ $labels.container }} in pod {{ $labels.pod }} located in namespace {{ $labels.namespace }} throttled for >1h with throttle of {{ humanizeDuration $value }}"
             expr: rate(container_cpu_cfs_throttled_seconds_total{container="postgres"}[5m]) > 0.1
           High_cpu_throttling:
-            enabled: true
+            enabled: false
```

Comment:
almost all the rules are needed by default, I really don't want to manually enable each one. What's the point?

Reply:
its very hard to understand what rule is enabled or not for an env. You need to check if the rule is enabled in charts and then in env. Because of this there where db rules enabled for int env (but theres no db in int) and redis rules where not enabled (although there is redis in int). So in my opinion disabling all rules by default and enabling only the ones that are needed in env gives us more visibility on what alerts we have